cmd/scale.go

```diff
@@ -123,20 +123,8 @@ func (sc *scaleCmd) load(cmd *cobra.Command) error {
 	sc.logger = log.New().WithField("source", "scaling command line")
 	var err error
 
-	if err = sc.authArgs.validateAuthArgs(); err != nil {
```

Comment:
@jackfrancis AzureStack flow would need load the environment information, so move the validation order.